crates/jstz_cli/src/config.rs

```diff
@@ -98,6 +99,18 @@ impl AccountConfig {
     pub fn list_all(&self) -> Vec<(&String, &Account)> {
         self.accounts.iter().collect()
     }
+
+    pub fn get_address(&self, address_or_alias: &str) -> Result<Address> {
+        if let Ok(address) = Address::from_base58(address_or_alias) {
+            return Ok(address);
+        }
+
+        if let Ok(account) = self.get(address_or_alias) {
+            return Ok(account.address().clone());
+        }
+
+        Err(anyhow!("Invalid smart function alias or address"))
```

Comment:
Since this is defined in AccountConfig, it would be better if the error messages don't mention smart functions, so that it can be more easily reused elsewhere.

```suggestion
        Err(anyhow!("Invalid alias or address"))
```

Reply:
nice catch!